### DISCUSSION TITLE
Disable IPTABLES Resolving

### DISCUSSION THREAD

**Discussion by shipler** [Security-Onion-Solutions/securityonion#1581] [Ideas] [answered]
setting a new IP for SO takes forever when there's no DNS resolving for certain hosts. 
Applying the following can skip the resolving time and make the process much quicker -

https://superuser.com/questions/1200224/how-to-disable-iptables-automatic-reverse-dns-lookup
> Include the -n option so it doesn't try to use DNS to resolve names for every ip address, network and port. Then it will be fast.

**Comment by TOoSmOotH** [comment, marked as answer]
You can't add rules with -n. Here is what I get when I try:

Try `iptables -h' or 'iptables --help' for more information.
[ERROR   ] Failed to set iptables rule for insert_INPUT_localhost_127.0.0.1_0:65535_udp.
Attempted rule was /sbin/iptables --wait -t filter -I INPUT 1 -p udp --dport 0:65535 --numeric --source 127.0.0.1 --jump ACCEPT
[ERROR   ] Command '['/sbin/iptables', '--wait', '-t', 'filter', '-I', 'INPUT', '1', '-p', 'tcp', '--dport', '4505', '--numeric', '--source', '127.0.0.1', '--jump', 'ACCEPT']' failed with return code: 2
[ERROR   ] stdout: iptables v1.4.21: Illegal option `-n' with this command
